examples/example_project/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+project(CCCLDemo CUDA)
+
+# This example uses the CMake Package Manager (CPM) to simplify fetching CCCL from GitHub
+# For more information, see https://github.com/cpm-cmake/CPM.cmake
+include(cmake/CPM.cmake)
+
+# This will automatically clone CCCL from GitHub and make the exported cmake targets available
+CPMAddPackage(
+    NAME CCCL
+    GITHUB_REPOSITORY nvidia/cccl
+    GIT_TAG monorepo # TODO: Replace this when monorepo is merged and release is out
+    OPTIONS "CCCL_ENABLE_LIBCUDACXX OFF" "CCCL_ENABLE_CUB OFF" "CCCL_ENABLE_THRUST OFF" "CCCL_ENABLE_TESTING OFF"
```

Comment:
I think we should default these options to `OFF`. We should optimize for making it easy for external users to use our cmake, and they don't care about enabling our tests. They just want our targets to link against.